tests/modules/nf-core/sourmash/index/nextflow.config

```diff
@@ -0,0 +1,9 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withName:SOURMASH_INDEX {
+        ext.args = "--ksize 31"
```

Comment:
Doesn't this need the `--dna`?

Reply:
Sorry, it was not needed, `--dna` is actually the default (I mixed up some errors previously).